nebula-exchange/src/main/scala/com/vesoft/nebula/exchange/reader/StreamingBaseReader.scala

```diff
@@ -36,18 +35,24 @@ class KafkaReader(override val session: SparkSession,
   require(kafkaConfig.server.trim.nonEmpty && kafkaConfig.topic.trim.nonEmpty && targetFields.nonEmpty)
 
   override def read(): DataFrame = {
+    import org.apache.spark.sql.functions._
     import session.implicits._
     val fields = targetFields.distinct
-    val jsonSchema = StructType(fields.map(field => StructField(field, StringType, true)))
-    session.readStream
-           .format("kafka")
-           .option("kafka.bootstrap.servers", kafkaConfig.server)
-           .option("subscribe", kafkaConfig.topic)
-           .load()
-           .selectExpr("CAST(value AS STRING)")
-           .as[(String)]
-           .withColumn("value", from_json(col("value"), jsonSchema))
-           .select("value.*")
+    val reader =
+      session.readStream
+        .format("kafka")
+        .option("kafka.bootstrap.servers", kafkaConfig.server)
+        .option("subscribe", kafkaConfig.topic)
+        .option("startingOffsets", kafkaConfig.startingOffsets)
+
+    val maxOffsetsPerTrigger = kafkaConfig.maxOffsetsPerTrigger
+    if(maxOffsetsPerTrigger.isDefined) reader.option("maxOffsetsPerTrigger", maxOffsetsPerTrigger.get)
+
+    reader.load()
+      .select($"value".cast(StringType))
+      .select($"value", json_tuple($"value", fields: _*))
+      .toDF("value" :: fields: _*)
```

Comment:
There's no need to keep the `value` field in the dataframe.

Reply:
I suggest keep it,if zhe data is invalid data,i will know source data to help fix bug

Reply:
there's no print operation for df, how can we know the source data's format? 
the way is to debug the data, then can we use `watch` to get the value data ?

Reply:
in Processor.scala filter will print it
i don't know watch means